scripts/Interface/ui/sans_isis/settings_diagnostic_tab.py

```diff
@@ -134,17 +134,17 @@ def fill_tree_widget(self, item, value):
             for key, val in sorted(value.iteritems()):
                 if key in self.excluded:
                     continue
-                child = QtGui.QTreeWidgetItem()
+                child = QtWidgets.QTreeWidgetItem()
```

Comment:
When trying to test whether this works I ran into an issue that the settings_diagnostic_tab does not work with python 3. The error is this 
`File "/home/matthew/Documents/Mantid/mantid/scripts/Interface/ui/sans_isis/settings_diagnostic_tab.py", line 134, in fill_tree_widget
    for key, val in sorted(value.iteritems()):`
`AttributeError: 'dict' object has no attribute 'iteritems'` which makes sense as iteritems no longer exists in python 3. Changing it to items instead should work and I don't think the size of the object is big enough to worry that this will be less memory efficient in python3. 

Could we possibly put this fix in here as it's minor or is that too much scope creep?

Reply:
I think it should be fine in this PR as it's a small change